clang/lib/Basic/Sanitizers.cpp

```diff
@@ -44,6 +92,19 @@ void clang::serializeSanitizerSet(SanitizerSet Set,
 #include "clang/Basic/Sanitizers.def"
 }
 
+void clang::serializeSanitizerMaskCutoffs(
+    const SanitizerMaskCutoffs &Cutoffs, SmallVectorImpl<std::string> &Values) {
+#define SANITIZER(NAME, ID)                                                    \
+  if (auto C = Cutoffs[SanitizerKind::SO_##ID]) {                              \
+    llvm::APFloat F(*C);                                                       \
```

Comment:
I don't like all format options.
Maybe better just to switch to int of (1/1000000), like we do in llvm?

Reply:
I agree floating-point is ugly (especially with imprecision), but a range of [1,1000000] feels too arbitrary. I suspect we'd get a lot of confused users putting in a value of 1 and getting the opposite result.

Reply:
Lets keep as is. If needed we can consider some form of fixed point parsing, printing, so output will look nicer.